src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -981,9 +981,6 @@ protected function getIndexSqlDefinition(Index $index, $tableName)
             $indexName = $index->getName();
         } else {
             $columnNames = $index->getColumns();
-            if (is_string($columnNames)) {
```

Comment:
Can someone confirm that removing this check is safe? 

Docblock of `\Phinx\Db\Table\Index::getColumns()` says it returns `array` and such a `is_string()` isn't present in `MysqlAdapter`, so I think removing this check here and in `SqlServerAdapter` shouldn't be a problem.